.dockerignore

```diff
@@ -1,15 +1,29 @@
 .gradle
 build
-cypress
+
+docs
+e2e
 nginx
+
 node_modules
-templates
 .dockerignore
+Dockerfile*
 .git
 .gitattributes
 .gitignore
-.gitlab-ci.yml
-Dockerfile*
+
 templates
 !templates/webpack
 !templates/nginx.conf
+
+# synchronize-marker:dist-scripts:begin
```

Comment:
Is there a tool behind this `synchronize-marker` ?

Reply:
Yes, searching for it will lead you to `scripts/check-file-synchronization.js`. 😉

That's also part of the pipeline:
https://github.com/intershop/intershop-pwa/blob/fd10846dc9775272dae94941dcc6dcf7424c4dc2/.github/workflows/development.yml#L72

Reply:
Yeah I found that file later on. Sorry for that.